README.md

```diff
@@ -10,34 +11,38 @@
 What is this?
 -------------
 
-Moodle comes with a built in 'security helper' which is what enforces the $CFG->curlsecurityblockedhosts are related settings. This plugin augments this existing security helper and adds additional features such as reporting on what urls are being curled to help inform policy decisions.
+Moodle comes with a built in 'security helper' which is what enforces the $CFG->curlsecurityblockedhosts and related settings. This plugin augments this existing security helper and adds additional features such as reporting on what urls are being curled to help inform policy decisions.
+
 
 How does it work?
 -----------------
 
 This relies on backporting this tracker:
 
-Curl Manger - Alternative security helper
+Allow plugins to augment the curl security helper via callback
+```
 https://tracker.moodle.org/browse/MDL-70649
+```
 
-In config.php specify the replacement security helper class:
+Please ensure a plugin function is defined as plugintype_pluginname_security_helper in pluginname/lib.php file.
```

Comment:
This doesn't make much sense here, this is really info useful to somebody writing a plugin which uses this callback. We already have this callback in place in this plugin.

Reply:
Made some changes please check again.